fetch-utils.js

```diff
@@ -0,0 +1,56 @@
+const SUPABASE_URL = 'https://sjdlsxmnlasfuesvmebz.supabase.co';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InNqZGxzeG1ubGFzZnVlc3ZtZWJ6Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDQzNDE0ODAsImV4cCI6MTk1OTkxNzQ4MH0.SGC6ShwKbSPLUcnWYRO3o7KQbJaV16eTy-AmGphLBh4';
+
+const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
+
+export async function createSticky(title, description, contact) {
+    const response = await client.from('Stickies').insert([
+        { Title: `${title}`,
+            Description : `${description}`,
+            Contact: `${contact}` }
+    ]);
+    console.log('createSticky', title, description, contact);
```

Comment:
Remember to delete those console.log's in the future, they can be tricky to find when building an application and looking for errors. Great work!